app/domain/output_document/html_renderer.rb

```diff
@@ -0,0 +1,51 @@
+class OutputDocument
+  class HTMLRenderer < SimpleDelegator
+    attr_accessor :output_document
+
+    def initialize(output_document)
+      @output_document = output_document
+      __setobj__(output_document)
+    end
+
+    def pages_to_render
+      case output_document.variant
+      when 'tailored'
+        circumstances = %i(continue_working unsure leave_inheritance
+                           wants_flexibility wants_security wants_lump_sum
+                           poor_health).select do |c|
+          output_document.public_send(c)
+        end
+
+        [:cover_letter, :introduction, circumstances, :other_information].flatten
```

Comment:
`%i(cover_letter introduction circumstances other_information)`

Thats the style we follow above and below.